Description: Include missing plugin mca_op_avx.so
Author:  Gilles Gouaillardet
Bug-Origin: https://github.com/open-mpi/ompi/issues/8323
Last-Updated: 2021-01-02
Forwarded: not-needed

Index: openmpi-4.1.0/ompi/mca/op/avx/Makefile.am
===================================================================
--- openmpi-4.1.0.orig/ompi/mca/op/avx/Makefile.am
+++ openmpi-4.1.0/ompi/mca/op/avx/Makefile.am
@@ -85,7 +85,7 @@ endif
 mcacomponentdir = $(ompilibdir)
 mcacomponent_LTLIBRARIES = $(component_install)
 mca_op_avx_la_SOURCES = $(sources)
-mca_op_avx_la_LIBADD = $(specialized_op_libs)
+mca_op_avx_la_LIBADD = $(specialized_op_libs) $(top_builddir)/ompi/lib@OMPI_LIBMPI_NAME@.la
 mca_op_avx_la_LDFLAGS = -module -avoid-version
 
 
